static/views/companies.html

```diff
@@ -453,7 +434,7 @@ <h4 style="margin-left: 9px;">Golang Cafe</h4>
 			alert('Please enter a valid email address');
 			return;
 		}
-		postMessage({email: emaile}, function(status) {
+		postMessage({email, message}, function(status) {
```

Comment:
don't think this ever worked 🤔

Reply:
good catch.. some of these pages have lots of copy pasted scripts. So just follow my PR comment I left. would be beneficial to use nested templates and split all re-usable html/css/js in sub-templates so we won't have to copy-paste the same stuff over and over again :)